tests/test_crs/valid_uperf_serviceip_nodeport.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       multus:
         enabled: false
       samples: 2
-      pair: 1
+      pair: 2
```

Comment:
I don't mind, but no other uperf test has multiple pairs, should we change that as well then? If so that should probably be in a  followup PR, thoughts?

Reply:
I would agree with this @smalleni

Reply:
Hey, just FYI, test scenarios are now specified within the e2e directory, uperf tests cases are specifically at https://github.com/cloud-bulldozer/benchmark-operator/tree/master/e2e/uperf
On the other hand, I agree to default to 2 pairs in the nodeport testcase